apps/o2-blocks/package.json

```diff
@@ -19,7 +19,7 @@
 	"scripts": {
 		"clean": "npx rimraf dist",
 		"prebuild": "npm run clean",
-		"build": "webpack --config='../../packages/calypso-build/webpack.config.js' --output-path='./dist' editor='./src/editor.js' "
+		"build": "webpack --config='../calypso-build/webpack.config.js' --output-path='./dist' --env.WP='true' editor='./src/editor.js' "
```

Comment:
Should prolly be `../packages/calypso-build` here

Reply:
😩 Everything got mixed up when I spotted the problem in #32138 

I think it should be in order now.